packages/cli/src/options/beaconNodeOptions/chain.ts

```diff
@@ -44,4 +46,12 @@ Will double processing times. Use only for debugging purposes.",
     description: "Persist invalid ssz objects or not for debugging purpose",
     group: "chain",
   },
+
+  "chain.proposerBoostEnabled": {
+    hidden: true,
```

Comment:
I think we can show this one, it's an important feature that will be switched on by default "soon"